packages/docs/pages/Alert/AlertPage.tsx

```diff
@@ -69,27 +69,44 @@ const AlertPage = () => (
       <Code>warning</Code>, <Code>success</Code>, <Code>info</Code>.
     </Text>
 
-    <Text>To use this component, wrap your app with this component:</Text>
+    <Text>
+      To use this component, put it in your root component (e.g. place it after <Code>GlobalStyles</Code> component):
+    </Text>
 
     <CodeSnippet>
-      {/* jsx-to-string:start */}
-      {function App() {
-        return (
-          <>
-            {/* ... */}
-            <AlertsManager manager={alertsManager} />
-            {/* ... */}
-          </>
-        );
-      }}
-
-      {/* jsx-to-string:end */}
+      {`
```

Comment:
We should prob investigate the extra `}` you are mentioning.

Reply:
My suggestion - jsx-to-string transformer works in a weird way, it depends on indentation in code, I moved two closing curly braces `}}` back, and it rendered it right. but there was a problem when you add just regular js code line before function.  it rendered it incorrectly, and also linter on commit hook was breaking formatting.